amundsen_application/static/js/components/DashboardPage/ShimmeringDashboardLoader/styles.scss

```diff
@@ -1,26 +1,33 @@
-@import 'variables';
+@import "variables";
 
 $loading-duration: 1s;
-$gradient-start-color: $gray5;
-$gradient-end-color: $gray15;
+$loading-curve: cubic-bezier(0.45, 0, 0.15, 1);
 $shimmer-border-radius: 4px;
 $shimmer-border-color: $gray20;
 $shimmer-height: 22vh;
 $shimmer-block-spacing: 16px;
 
 @keyframes shimmer {
   0% {
-    background-position: -1000px 0;
+    background-position: 100% 0;
   }
   100% {
-    background-position: 1000px 0;
+    background-position: 0 0;
   }
 }
 
 .animate {
```

Comment:
Looks like this would apply to any element with the "animate" class. Should we rename this or add some additional selector rules to restrict this?

Reply:
I see, yeah, it could be a too common class name. 
What about `is-animated`? Putting it as a "state" on a SMACSS way.